client/package.json

```diff
@@ -6,12 +6,12 @@
     "@hot-loader/react-dom": "16.12.0",
     "@material-ui/core": "^4.11.0",
     "bowser": "^2.9.0",
-    "cross-fetch": "^3.0.4",
+    "cross-fetch": "^3.1.4",
```

Comment:
Hey @swillrich5 , curious about these changes. Typically we do not change our committed versions of packages unless there is a need.

Reply:
The only thing I can think of there is that I ran "npm audit fix" after "npm install".  Maybe that caused the update?  I can understand why that would be a concern.

Reply:
hi @swillrich5 did these package versions still need to get rolled back or are the new versions what we're sticking with?

Reply:
In talking with Sal, he was OK with the new package version, but told me to watch out for it in the future.